examples/simple/main.go

```diff
@@ -35,6 +35,12 @@ func gridDemo(port int32) *monome.Grid {
 					if ke.S == int32(1) {
 						g.LedSet(ke.X, ke.Y, ke.S)
 					}
+				case monome.TiltEvent:
+					te := e.(monome.TiltEvent)
+					fmt.Println(te.N)
```

Comment:
Since we have a `String` method on `TiltEvent` we could just print that instead of the individual print statements?
```suggestion
					fmt.Println(te)
```

Reply:
absolutely agree with these changes, have added another commit!